lib/purescript-cst/README.md

```diff
@@ -1,12 +1,13 @@
 # purescript-cst
 
-Defines the surface syntax of the PureScript Programming Language.
+Defines the surface and underlying syntax of the PureScript Programming Language.
```

Comment:
Now that these packages are merged, I think talking about a distinction between a "surface" and "underlying" syntax is probably unnecessary, and I suspect it won't be clear to lots of readers, and they probably don't need to care about the difference in lots of cases anyway, at least not immediately. How about "The parser for the PureScript programming language"?

Reply:
> How about "The parser for the PureScript programming language"?

Isn't this more than just the parser? Does one ever operate directly on the AST via some tool?

I'll make the change, but I'm just expressing my confusion as to why `syntax` is no longer mentioned when it seems like that's what this resulting library is.

Reply:
It has type definitions for the syntax, sure, but you can’t really write a parser without having those in the first place. Maybe I’m overthinking it though. I guess “defines the syntax for …” would work too.

Reply:
I think `syntax and parser` would probably work, but... :upside_down_face: I already merged this. 

We can fix it later if someone gets confused down the road.